helm/flowforge/templates/broker.yaml

```
@@ -19,7 +19,7 @@ data:
     auth_opt_hasher bcrypt
     auth_opt_cache true
     auth_opt_auth_cache_seconds 30
```

```suggestion
    auth_opt_auth_cache_seconds 120
```

Suggest increasing this as well